static/app/views/performance/onboarding/useHasData.tsx

```diff
@@ -0,0 +1,33 @@
+import type {PageFilters} from 'sentry/types';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {useSpanMetrics} from 'sentry/views/starfish/queries/useDiscover';
+
+export function useHasData(filter: string, referrer: string) {
```

Comment:
For consistency, i like the idea of `filter` being of type `MutableSearch` to match the api of `useSpanMetrics` better.